backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -88,8 +103,45 @@ const parseWebhook = async (
   throw new UnexpectedWebhookError('Unexpected webhook body')
 }
 
+const sendPasscodeCreationMessage = async (
+  whatsappId: WhatsAppId,
+  clientId: WhatsAppApiClient
+): Promise<MessageId> => {
+  const templateMessageToSend: WhatsAppTemplateMessageToSend = {
+    recipient: whatsappId,
+    apiClient: clientId,
+    templateName: 'sgc_passcode_generation', // TODO: Un-hardcode this
```

Comment:
Could I ask why we expect this value to change?

Reply:
Whenever we want to use a different template to enable passcode creation. This can happen on a whim, so it'll be ideal to not have to make a commit to change this value.

Also, when we update a template, it remains unavailable until re-approved. As for how long it takes for a template to be re-approved, it took 1 working day for the prototyping WABA and I am unsure how long it takes for the production WABA.

Will love to address this TODO in the next PR. Let me make a Linear task for this.

Reply:
Sure, we can discuss the implementation details for this when we embark on this task.